kolibri/plugins/app/api.py

```diff
@@ -32,25 +23,23 @@ def has_permission(self, request, view):
 
 class AppCommandsViewset(ViewSet):
 
-    permission_classes = (FromSameDevicePermission, FromAppViewPermission)
+    permission_classes = (FromAppViewPermission,)
```

Comment:
Should we also make this `FromAppContextPermission`, for full consistency?

Reply:
(or just `AppContextPermission`)

Reply:
Sure